ROADMAP.md

```diff
@@ -30,35 +30,26 @@ Status updates
 Short-term goals
 ----------------
 
-These are goals achievable within 3-6 months, tentatively done by Q3 2022.
+These are goals achievable within 3-6 months, tentatively done by Q2 2023.
 
-- **Stabilize the existing functionality and fix any major, high-impact, and blocking bugs.**<br>
-  Tool stability is something we take very seriously and users shouldn't run into any major showstopping issues.
+- **Make xk6-browser a part of standard distribution of k6.**<br>
 
   *How will we achieve this?*<br>
-  By manually testing the application in different scenarios, adding more automated tests, and addressing reports from internal and external users.
-  Any issues found that have a considerable user impact will be given highest priority and fixed ASAP.
+  xk6-browser will become an [experimental module](https://k6.io/docs/javascript-api/k6-experimental/) of k6, and thus available to use in load tests that utilize standard k6 release builds.
 
   *Definition of Done*<br>
-  This is difficult to gauge, but at some point we should stop running into any major issues on a frequent basis and should consider this goal as reached.
+  A load test script that imports `k6/x/browser` can be executed using the latest release build of k6. The browser-level APIs provided by xk6-browser are available for use within this script.
 
-
-- **Limited Alpha deployment and testing of the extension in k6 Cloud.**<br>
```

Comment:
I think the private beta should still be part of the roadmap as we've not officially onboarded anyone external from k6 (or even Grafana).

Reply:
But how would that work since k6 will use k6-browser (after the nearing merge)? I mean, k6-browser will be available for all cloud users. Although, we can find ways to tweak this behavior, like, not releasing it to some users.

Reply:
@ankur22 which does not mean that we cannot onboard them any time at the moment, right? 😉